easybuild/easyblocks/generic/cmakemake.py

```diff
@@ -83,6 +95,10 @@ def configure_step(self, srcdir=None, builddir=None):
                 srcdir = default_srcdir
 
         options = ['-DCMAKE_INSTALL_PREFIX=%s' % self.installdir]
+
+        if '-DCMAKE_BUILD_TYPE' not in self.cfg['configopts']:
+            options.append("-DCMAKE_BUILD_TYPE=%s" % self.cfg['buildtype'])
```

Comment:
no, because the other configure options don't have a matching easyconfig parameter that can be set?

the problem is that you can specify the value for `-DCMAKE_BUILD_TYPE` in two ways now, it should be clear from the logs which one wins